components/analytics/usage-history.tsx

```diff
@@ -102,39 +94,33 @@ export function UsageHistory() {
           })
       );
     })();
-  }, [settings.loggedIn]);
+  }, []);
   return (
     <Card>
       <CardHeader>
         <CardTitle className="text-sm font-medium">Usage History</CardTitle>
       </CardHeader>
       <CardContent>
-        {usageHistory ? (
```

Comment:
Don't delete the changes I made. They show Loading when the usage history is being loaded (showing `LoadingCircle` component)